combine/stat.go

```diff
@@ -0,0 +1,13 @@
+package combine
+
+import "os"
+
+// concatenateAndCompressStatsFiles finds and conatenates the stats files and
+// compresses the output.
+func concatenateAndCompressStatsFiles(inputs []*os.File, output *os.File) error {
```

Comment:
Make public and use in cmd/combine.go.

Consider renaming all the file-type specific public functions to something simple like "StatFiles()", so you call it as combine.StatFiles(inputs, output). The docs can explain that concatenation and compression is happening.